src/averageAges.js

```diff
@@ -14,51 +14,55 @@
  *
  * @return {number}
  */
+
+const sexMan = (people) => people.filter(person => person.sex === 'm');
+const sexWoman = (people) => people.filter(person => person.sex === 'f');
```

Comment:
You could create a single function for it, and decide which group it should filter out basing on a param.
```suggestion
const sexFilter = (people, sex) => people.filter(person => person.sex === sex);
```